opensrp/fhircore
# Quest: Add ability view QuestionnaireResponse

**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #679

**Type**
Choose one: Feature

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app 


### To test
1. Run `quest` app
2. Click on **Isaya Mollel** and click on the 25th Oct 2021 G6PD results. This should open the form activity with all the questions as read-only and a **DONE** button
